lib/scooter/httpdispatchers/orchestrator/v1/v1.rb

```diff
@@ -89,6 +89,21 @@ def post_inventory(payload)
         def get_status
           @connection.get("#{@version}/status")
         end
+
+        #task endpoints
+        def get_task(mod, task)
+          @connection.get("#{@version}/tasks/#{mod}/#{task}")
+        end
+
+        def get_tasks
+          @connection.get("#{@version}/tasks")
+        end
+
+        def run_task(payload)
```

Comment:
I think run_task makes more sense, and want to gravitate towards descriptive rather than "post" and "get" anyhow, but if you think post_task is better I can change it. It's definitely more consistent with the rest of the Scooter.